tests/covalent_dispatcher_tests/choose_conda_test.py

```diff
@@ -39,7 +39,7 @@ def test_using_current_env() -> None:
     tmp_executor.get_conda_envs()
     conda_env = tmp_executor.current_env
 
-    executor = ct.executor.LocalExecutor(conda_env=conda_env, current_env_on_conda_fail=True)
+    executor = ct.executor.LocalExecutor(conda_env=conda_env, current_env_on_conda_fail=False)
```

Comment:
Why are we changing this?

Reply:
With `current_env_on_conda_fail=True`, it will successfully run, even if something goes wrong when trying to run on a different Conda environment.